ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml

```diff
@@ -236,6 +236,20 @@ tests:
     test:
     - ref: openshift-qe-router-perf
     workflow: rosa-aws-sts-hypershift
+- as: perfscale-ci-tests-rosa-ingress-perf-3nodes
+  cron: 0 7 * * 1
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      REPLICAS: "3"
```

Comment:
Same question here, REPLICAS=3? Can you ptal at Sachin's and Joe's comments here https://redhat-internal.slack.com/archives/GTHU29259/p1688584111328949
I am not sure what the conclusion (if any) there is.

Reply:
@svetsa-rh This pr is for ingress-perf, Sachin and Joe's comments should be for k8s-netperf.

Reply:
@svetsa-rh I'm ok with this PR, I'll leave it to you to add lgtm.